src/Riok.Mapperly/Diagnostics/DiagnosticDescriptors.cs

```diff
@@ -96,8 +96,8 @@ internal static class DiagnosticDescriptors
 
     public static readonly DiagnosticDescriptor SourcePropertyNotFound = new DiagnosticDescriptor(
         "RMG012",
-        "Mapping source property not found",
-        "Property {0} on source type {1} was not found",
+        "Source property was not found for target property",
+        "The property {0} on the mapping target type {1} was not found on the mapping source type {2}",
```

Comment:
My english is not too good, but I'm not sure about this sentence. It says the property was not found on the target type and not on the source type 🤔 But it is actually only missing on the source type, isn't it?
What about:
`While mapping to {0} on {1} no property named {0} could be found on the source type {1}`

and 
`While mapping to the required property {0} on {1} no property named {0} could be found on the source type {1}`
for `RMG023`?

Reply:
It seems like your English is great. I'm a native speaker and it's confusing. There are two existing messages which I was attempting to parallel.

* RMG020: "The property {0} on the mapping source type {1} is not mapped to any property on the mapping target type {2}"
* RMG023: "Required property {0} on mapping target type was not found on the mapping source type {1}"

These are the updated versions (23 now includes the target type):
* RMG020: "The property {0} on the mapping source type {1} is not mapped to any property on the mapping target type {2}"
* RMG023: "Required property {0} on mapping target type {1} was not found on the mapping source type {2}"

This change updates 12 so that it's a combination of 20 and 23:
* RMG012: "The property {0} on the mapping target type {1} was not found on the mapping source type {2}"

I'd be happy to switch it around if you prefer. In that case, we might want to update 20 to keep it consistent.

Reply:
Ah nvm, now I understand it.